cpp/include/cudf/stream_compaction.hpp

```diff
@@ -259,5 +259,19 @@ cudf::size_type unique_count(column_view const& input,
                              null_policy null_handling,
                              nan_policy nan_handling);
 
+/**
+ * @brief Count the unique elements in the table_view
+ *
+ * Given an input table_view, number of unique rows in this table_view is returned
+ *
+ * @param[in] input The column_view whose unique elements will be counted.
+ * @param[in] nulls_equal flag to denote nulls are equal if null_equality::EQUAL,
```

Comment:
No need to repeat the meaning of `null_equality::EQUAL/UNEQUAL`. 
```suggestion
 * @param[in] nulls_equal flag to denote if null elements should be considered equal
```